data/World/America/USA/ncdot.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "name": "NCDOT",
+        "description": "North Carolina CORS",
+        "urls": [
+            "http://207.4.96.200:2101",
+            "http://rtn.nc.gov:2101"
+        ],
+        "reference": {
+            "url": "https://ncgs.state.nc.us/pages/CORS-and-GNSS.htm#:~:text=The%20NC%20CORS%20Base%20Station%20Network%20is%20referenced%20to%20the%20NAD%2083(2011)%20datum."
+        },
+        "last_update": "2025-04-11",
+        "streams": [
+            {
+                "filter": "all",
```

Comment:
I don't love using `all` here since it'll cause issues when NSRS is released and starts being used, but the operator was not very responsive via email, and this is accurate as of today.

Reply:
Update: Curt just got back to me with a list of their mountpoints, and I'll update them in here. I've attached a PDF of the email to the PR description.